src/corehost/build.proj

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
We are going to need to be able to build the host without msbuild in the picture. Looks like the majority of what you are doing here is creating a command line and generating the versioning stuff, we may need to figure out a way to do that purely from the script files.